### PR TITLE
Candidate fix for #118

### PR DESCRIPTION
**Pull request by maksymiuks**
PR related to #118

### CODE REVIEW

File: R/plot_feature_importance.R [hunk 1]
```diff
@@ -151,12 +151,16 @@ plot.feature_importance_explainer <- function(x, ..., max_vars = NULL, show_boxp
                    stat = "identity", fill = "#371ea3", color = "#371ea3", width = 0.25)
   }
 
+  if (!is.null(attr(x, "loss_name"))) {
+    y_lab <- paste(attr(x, "loss_name"), "loss after permutations")
```
Comment (pbiecek) [inline R/plot_feature_importance.R]: make sure that this works if there is a vector in attr(x, "loss_name").
Maybe use `attr(x, "loss_name")[1]`

File: R/plot_feature_importance.R [hunk 2]
```diff
@@ -151,12 +151,16 @@ plot.feature_importance_explainer <- function(x, ..., max_vars = NULL, show_boxp
...
+  } else {
+    y_lab <- "Drop-out loss after permutations"
```
Comment (pbiecek) [inline R/plot_feature_importance.R]: Maybe `"Loss function after variable's permutations"`

File: R/plot_feature_importance.R (hunk 1 above)
Comment (maksymiuks) [reply R/plot_feature_importance.R]: Done

File: R/plot_feature_importance.R (hunk 2 above)
Comment (maksymiuks) [reply R/plot_feature_importance.R]: Done

**Pull request merged**